### DISCUSSION TITLE
Move Cursor to Line Start

### DISCUSSION THREAD

**Discussion by waveskimmer** [dbeaver/dbeaver#22040] [Q&A]
Is `Line Start` the correct keybinding?   As I can't seem to get it working.

![LineStart](https://github.com/dbeaver/dbeaver/assets/31574609/79dc3662-988a-4519-9a4b-a2a3c21e51cf)

**Comment by LonwoLonwo** [comment]
Hello @waveskimmer 

Where do you try to use it?
